src/Design.cpp

```diff
@@ -0,0 +1,205 @@
+#include "Design.hpp"
+
+namespace {
+	// private
+	struct CGaugeValue : public SokuLib::CGauge::IValue {
```

Comment:
I don't use the legacy code naming scheme because it is not very IDE friendly so remove the C at the start of classes please. For The interface you can leave the I before.
You can leave the legacy code name as a comment above though.